src/main/java/net/coderbot/iris/mixin/MixinTitleScreen.java

```diff
@@ -0,0 +1,25 @@
+package net.coderbot.iris.mixin;
+
+import net.coderbot.iris.Iris;
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.client.gui.screen.FatalErrorScreen;
+import net.minecraft.client.gui.screen.NoticeScreen;
+import net.minecraft.client.gui.screen.TitleScreen;
+import net.minecraft.text.LiteralText;
+import net.minecraft.util.Formatting;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+@Mixin(TitleScreen.class)
+public class MixinTitleScreen {
+	@Inject(method = "init", at = @At("RETURN"))
+	public void showError(CallbackInfo ci) {
```

Comment:
Prefix with `iris$` and maybe make the inject method name more descriptive, i.e. "showSodiumIncompatScreen"

Reply:
sounds good, will do this

Reply:
Done.